easybuild/easyconfigs/n/neko/neko-0.9.1-foss-2025a-cpu.eb

```
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'neko'
+version = '0.9.1'
+versionsuffix = '-cpu'
+
+toolchain = {'name': 'foss', 'version': '2025a'}
+
+homepage = 'https://neko.cfd/'
+description = """Neko is a portable framework for high-order spectral element flow simulations."""
+
+source_urls = ['https://github.com/ExtremeFLOW/neko/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [
+    ('json-fortran', '8.3.0'),
```

```suggestion
    ('json-fortran', '8.3.0'),
    ('HDF5', '1.14.6'),
    ('ParMETIS', '4.0.3'),
```

Lets include these two dependencies as well. HDF5 is marked as highly recommended, ParMETIS at least as optional.